tests/after_cluster_deploy_sanity/test_after_cluster_deploy_sanity.py

```diff
@@ -164,3 +164,10 @@ def test_csv_health(admin_client, hco_namespace):
         timeout=TIMEOUT_5MIN,
         stop_status="fakestatus",  # to bypass intermittent FAILED status that is not permanent.
     )
+
+
+@pytest.mark.cluster_health_check
+def test_common_node_cpu_model(cluster_node_cpus, cluster_common_node_cpu, cluster_common_modern_node_cpu):
+    assert cluster_common_node_cpu and cluster_common_modern_node_cpu, (
+        f"This is a heterogeneous cluster: {cluster_node_cpus}"
```

Comment:
Heterogeneous Clusters are perfectly fine to have.

We need to mark the cluster as unhealthy, only when

1. Cluster is Heterogeneous and
2.  If we do not find even a single common CPUModel.

Reply:
Maybe I should have asked first, so a question?  Is this check, already checking for "atleast one common cpu model" on the cluster ?

Reply:
If so, can you please update the comment to "This is a heterogeneous cluster, with no common CPU Model: ..." ?

Reply:
+1 for adding "with no common CPU Model"

Reply:
Will update the message

Reply:
Done.